util-images/watch-list/main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+	v1 "k8s.io/client-go/informers/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog/v2"
+)
+
+func main() {
+	var kubeconfig string
+	var targetNamespace string
+	var informerCount int
+	var testTimeout time.Duration
+
+	klog.InitFlags(flag.CommandLine)
+	flag.StringVar(&kubeconfig, "kubeconfig", "", "path to kubeconfig.")
+	flag.StringVar(&targetNamespace, "namespace", "huge-secrets-0", "namespace that host secrets to list. If empty a default (huge-secrets-0) value will be used.")
+	flag.IntVar(&informerCount, "count", 4, "the number of informers per targetNamespace to run. If empty a default (4) value will be used.")
+	flag.DurationVar(&testTimeout, "timeout", time.Minute, "timeout duration for the test")
```

Comment:
I need to figure out the timeout for the test. I'd start with 5 min or so.

Reply:
I think this needs to be infinity given that in CL2 config you kill the pod after some time.